ci/kokoro/docker/define-docker-variables.sh

```diff
@@ -16,6 +16,15 @@
 
 set -eu
 
+if [[ -z "${NCPU+x}" ]]; then
+  NCPU=$(nproc)
+  # Mac doesn't have nproc. Run the equivalent.
+  if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
If we're on a mac, won't the previous `nproc` command fail and cause the script to exit?

Reply:
Good point. Thanks.